packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1843,10 +1843,19 @@ export class ProjectService {
             description: 'Gets all filters available for a single query',
         });
         try {
-            const savedChart = await this.savedChartModel.get(savedChartUuid);
+            const savedChart =
+                await this.savedChartModel.getInfoForAvailableFilters(
+                    savedChartUuid,
+                );
 
             if (
-                user.ability.cannot('view', subject('SavedChart', savedChart))
+                user.ability.cannot(
+                    'view',
+                    subject('SavedChart', {
+                        ...savedChart,
+                        organizationUuid: user.organizationUuid,
```

Comment:
We should use the org id returned from the db. Problem with this change:
- Admin from org A requests chart from org B

Reply:
Ok that's good to know! thank you, will check this out now